django/utils/version.py

```diff
@@ -76,6 +76,11 @@ def get_git_changeset():
     This value isn't guaranteed to be unique, but collisions are very unlikely,
     so it's sufficient for generating the development version numbers.
     """
+    # If __file__ is undefined in this module, we won't be able to find the
+    # repo. Besides, we're likely running inside a frozen module, which isn't
+    # necessarily frozen from the repo's working tree in the first place.
```

Comment:
See [the coding style guide here](https://docs.djangoproject.com/en/3.1/internals/contributing/writing-code/coding-style/#python-style). Avoid 'we' in comments. 

Maybe 

```suggestion
    # Repo can't be found if __file__ is undefined, e.g. in a frozen module.
```